service/src/main/java/com/ge/predix/acs/policy/evaluation/cache/AbstractPolicyEvaluationCache.java

```diff
@@ -162,6 +162,13 @@ public void resetForResource(final String zoneId, final String resourceId) {
     }
 
     @Override
+    public void resetForResourcesByIds(final String zoneId, final Set<String> resourceIds) {
+        for (String resourceId : resourceIds) {
```

Comment:
This approach is wasteful when using a Redis as a cache. Instead, consider pipelining the requests to reset the id so that there is only one latency penalty. See [here](https://github.com/predix/acs/blob/de3b2a7b1338b7cca73904cc839b19d718452686/service/src/main/java/com/ge/predix/acs/policy/evaluation/cache/RedisPolicyEvaluationCache.java#L130-L142) as an example of how to pipeline requests to Redis.